client/data/site-profiler/types.ts

```diff
@@ -93,6 +93,7 @@ export interface HostingProviderQueryResponse {
 }
 
 export type Metrics = 'cls' | 'fid' | 'lcp' | 'fcp' | 'ttfb' | 'inp';
+export type NewMetrics = Exclude< Metrics, 'fid' >;
```

Comment:
I had to add this new type, because the existing `Metrics` type includes `fid`, so it's expected to include it in every mapping we use (dictionaries, API endpoint return types, etc.). I tried removing `fid` from the `Metrics` type, but that broke typecheck for a lot of files in the `site-profiler` folder, so overall this seemed the easier solution. @WBerredo do yo have some tip perhaps on how best to go about this?

Reply:
What I've done in https://github.com/Automattic/wp-calypso/pull/93543/files was to change the type anyway and on the old site profiler files I left it as `any`. I don't think its a big deal considering we plan to remove it in the near future.

Reply:
Oh that seems like a better workaround! Let me pull trunk and try to do the same here.